bridge/pkg/terra/terra.go/client/broadcast.go

```diff
@@ -0,0 +1,66 @@
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/certusone/wormhole/bridge/pkg/terra/terra.go/msg"
+	"github.com/certusone/wormhole/bridge/pkg/terra/terra.go/tx"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// BroadcastReq broadcast request body
+type BroadcastReq struct {
+	Tx   tx.StdTxData `json:"tx"`
+	Mode string       `json:"mode"`
+}
+
+// TxResponse response
+type TxResponse struct {
+	Height msg.Int `json:"height"`
+	TxHash string  `json:"txhash"`
+	Code   uint32  `json:"code,omitempty"`
+	RawLog string  `json:"raw_log,omitempty"`
+}
+
+// Broadcast - no-lint
```

Comment:
This is terra library terra.go added as sources (since it is not published yet). Once it is published we're going to replace the whole `terra.go` dir with an import statement.